Bug fixes

akempes opened the pull request:
Hi, I really like the 'lightweightness' of this script. 
But the class 'dropping-above' would never be added to an element due to a bug in the if-statement. That's fixed now.

I also made it possible to use a list in a nested object. (Like drag-to-reorder="myData.myListToReorder")

Thanks!

brandly commented:
thanks for the help! at a glance, this code looks nice, but i still need to pull it down and test things out a bit.

i've been slacking a bit, so i haven't gotten around to #1. we'll probably run into some conflicts, but i'll get these merged fairly soon.

akempes commented:
I added two events, dragstart and dragend. I'm using these events to connect to my server and store the changes. Dragstart will abort the request made when the dragend event was fired. I added a demo as well to make clear how it works...
